clients/go/test/grpc-app/main.go

```diff
@@ -42,7 +42,7 @@ func main() {
 		syscall.SIGINT, syscall.SIGTERM)
 	defer done()
 
-	logger := logging.NewFromEnv("")
+	logger := logging.New(os.Stdout, logging.LevelDebug, logging.FormatJSON, true)
```

Comment:
⚠️ Behavior change - the shell app seems to never have been configured via the environment. I enabled the lowest level of debugging by default.